### PR TITLE
Feature/tt 1130 admin import

### PR DESCRIPTION
**Pull request by sarahentzel**

### CODE REVIEW

File: src/store/importexport/actions.tsx [hunk 1]
```diff
@@ -40,36 +43,139 @@ export const exportProject = (
...
+    API_CONFIG.host + '/api/offlineData/project/import/' + files[0].name;
+  Axios.get(url, {
+    headers: {
+      Authorization: 'Bearer ' + auth.accessToken,
+    },
+  })
+    .then(response => {
+      const filename = response.data.data.attributes.message;
+      const xhr = new XMLHttpRequest();
+      xhr.open('PUT', response.data.data.attributes.fileurl, true);
```
Comment (gtryus) [inline src/store/importexport/actions.tsx]: How large a file can this handle? If we start adding audio, will we have to refactor this?

File: src/store/importexport/electronExport.tsx [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+        id: '1',
+      },
+    };
+  };
+
+  const AddCheckEntry = (): string => {
+    var dt = new Date().toUTCString();
+    zip.addFile(
+      'SILTranscriberOffline',
+      Buffer.alloc(dt.length, dt),
```
Comment (gtryus) [inline src/store/importexport/electronExport.tsx]: Will these alloc's cause memory leaks? Where does this memory get freed up again?

File: src/store/importexport/electronExport.tsx [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+          contenttype: 'application/zip',
+        },
+        type: 'file-responses',
+        id: '1',
+      },
+    };
+  };
+
+  const AddCheckEntry = (): string => {
+    var dt = new Date().toUTCString();
```
Comment (gtryus) [inline src/store/importexport/electronExport.tsx]: Do we want to use the date function we made or is this date in a different format? They both seem to be utc dates...

Comment (gtryus) [inline src/store/importexport/electronExport.tsx]: Do we want to use the date function we made or is this date in a different format? They both seem to be utc dates...

File: src/store/importexport/electronExport.tsx (hunk 2 above)
Comment (sarahentzel) [reply src/store/importexport/electronExport.tsx]: I didn't find anything about an unalloc so I'm assuming they go away at function end.

File: src/store/importexport/actions.tsx (hunk 1 above)
Comment (sarahentzel) [reply src/store/importexport/actions.tsx]: This is the same s3Put link that we use for audios.  Limit is 5G, but it's recommended to use a multipart upload > 100M

File: src/store/importexport/electronExport.tsx (hunk 3 above)
Comment (sarahentzel) [reply src/store/importexport/electronExport.tsx]: yes!

**Pull request merged**